services/movie_session.py

```diff
@@ -42,3 +42,10 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list:
+    return [
+        {"row": tiket.row, "seat": tiket.seat}
```

Comment:
There is a typo in the `get_taken_seats` function: `tiket` should be corrected to `ticket` to match the variable name used in the loop.